.github/workflows/reusable-release.yml

```diff
@@ -45,51 +45,19 @@ jobs:
     name: 🔒 Init
     runs-on: ubuntu-latest
     outputs:
-      key_fingerprint: ${{ steps.read-identifiers.outputs.key_fingerprint }}
-      key_email: ${{ steps.read-identifiers.outputs.key_email }}
-      release_type: ${{ steps.generate-version-suffix.outputs.release_type }}
-      version_suffix: ${{ steps.generate-version-suffix.outputs.version_suffix }}
+      key_fingerprint: ${{ steps.init.outputs.key_fingerprint }}
+      key_email: ${{ steps.init.outputs.key_email }}
+      release_type: ${{ steps.init.outputs.release_type }}
+      version_suffix: ${{ steps.init.outputs.version_suffix }}
     steps:
-      - uses: actions/setup-java@v4
+      - uses: actions/checkout@v4
+        with:
+          repository: guardian/gha-scala-library-release-workflow # TODO - adapt for forks
+          ref: break-workflow-up-into-smaller-reusable-files
+      - id: init
+        uses: ./.github/actions/init
```

Comment:
Should we check this out to a separate `path` so it doesn't mix up with the code of the repo where the action may be running?

Reply:
:laugh-cry: